tests/rule-engine/bc-modus-ponens.scm

```diff
@@ -0,0 +1,56 @@
+; =============================================================================
+; Modus Ponens Rule
+;
+; Given P(A implies B) and sA, calculate sB
+; -----------------------------------------------------------------------------
+
+(define pln-rule-modus-ponens
+    (BindLink
+        (VariableList
+            (VariableNode "$A")
+            (VariableNode "$B"))
+        (ImplicationLink
+            (AndLink
+                (ImplicationLink
+                    (VariableNode "$A")
+                    (VariableNode "$B"))
+                (VariableNode "$A"))
+            (ExecutionOutputLink
+                (GroundedSchemaNode "scm: pln-formula-simple-modus-ponens")
+                (ListLink
+                    (VariableNode "$B")
+                    (ImplicationLink
+                        (VariableNode "$A")
+                        (VariableNode "$B")))))))
+
```

Comment:
Yeah, I was trying to make the tests more self a contained unit.  I didn't do it uniformly enough yet though as other places still refer to the examples folder.